### PR TITLE
FWH-67 Audio Message contentType: 8 handling | Web SDK

### PR DESCRIPTION
**Pull request by pranav4-4**
### What do you want to achieve?
- when we send audio from mobile it should be handled in web sdk

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [x] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

- I have tested it from https://test.prochnost.tk/widget-test

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

**Comment by prochnost04** [comment]
@pranav4-4 I guess you have mistakenly formatted whole file.

I would suggest you to undo the formatting and then file a PR.

File: webplugin/js/app/kommunicate.js [hunk 1]
```diff
@@ -493,7 +493,8 @@ $applozic.extend(true, Kommunicate, {
         return (
             (typeof msg.fileMeta === 'object' &&
                 msg.contentType ==
-                    KommunicateConstants.MESSAGE_CONTENT_TYPE.ATTACHMENT) ||
+                    KommunicateConstants.MESSAGE_CONTENT_TYPE.ATTACHMENT) || 
+                    msg.contentType == 8 ||
```
Comment (prochnost04) [inline webplugin/js/app/kommunicate.js:497]: make 8 a constant

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 2]
```diff
@@ -8804,7 +8804,7 @@ var userOverride = {
                         ? 'km-custom-widget-background-color'
                         : 'km-custom-widget-background-color-secondary';
 
-                if ( floatWhere !== 'mck-msg-right' && kmAttchMsg === 'km-attach-msg') {
+                if ( (floatWhere !== 'mck-msg-right' && kmAttchMsg === 'km-attach-msg') || msg.contentType == 8) {
```
Comment (prochnost04) [inline webplugin/js/app/mck-sidebox-1.0.js:8807]: check how kmAttchMsg variable is getting set
we need to make the change there

**Review by prochnost04 — CHANGES_REQUESTED** [review]

File: webplugin/js/app/mck-sidebox-1.0.js (hunk 2 above)
Comment (pranav4-4) [reply webplugin/js/app/mck-sidebox-1.0.js]: done

File: webplugin/js/app/kommunicate.js (hunk 1 above)
Comment (pranav4-4) [reply webplugin/js/app/kommunicate.js]: done

**Review by prochnost04 — DISMISSED** [review]

**Comment by pranav4-4** [comment]
* Created a new branch for developent branch and merged it

**Pull request closed without merging**